docs/source/building_applications/rag.md

```diff
@@ -162,3 +162,8 @@ for vector_db_id in client.vector_dbs.list():
     print(f"Unregistering vector database: {vector_db_id.identifier}")
     client.vector_dbs.unregister(vector_db_id=vector_db_id.identifier)
 ```
+
+### Supported Data Types
+
+You can ingest data into the vector database using `RAGDocument` objects.
+The supported mime types (https://en.wikipedia.org/wiki/Media_type#mime.types) data types are:
```

Comment:
Was the plan to list the actual mime types supported?

Reply:
Oops yeah let me take this out for now. That is in the to do list to document RAG more formally.

Reply:
Removed, thanks for catching it!